json_serializable/CHANGELOG.md

```diff
@@ -17,6 +17,10 @@
 
 ### `package:json_serializable/type_helper.dart`
 
+* *Breaking* The `nullable` parameter on `TypeHelper.serialize` and 
```

Comment:
Other breaking callouts use `**` for bold instead of `*` for italics

Reply:
fixed